### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by C0rn3j**
Change the cloning steps (current do not work for neither paths nor regular users as they do not have git access to this repository).

Make the readme a bit more consistent, fix a typo or two, add some syntax highlighting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
...
+```
+# apt-get install icinga2-slack-notifications
+```
 
 This will create the plugin files in the correct `icinga2` conf directory. 
 
 ### Installation using git
 
-1. clone the repository under your Icinga2 `/etc/icinga2/conf.d` directory
+1. Clone the repository under `/opt` and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory
```
Comment (nisabek) [inline README.md:88]: does it matter where it's cloned in the beginning?...

File: README.md [hunk 2]
```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
...
 here are the steps to perform:
 
-```
-apt-get install -y apt-transport-https
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
-add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
-apt-get update
+```bash
+# apt-get install -y apt-transport-https
+# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
```
Comment (nisabek) [inline README.md:73]: why the "#" in the beginning of the line?..

File: README.md [hunk 3]
```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
...
 This will create the plugin files in the correct `icinga2` conf directory. 
 
 ### Installation using git
 
-1. clone the repository under your Icinga2 `/etc/icinga2/conf.d` directory
+1. Clone the repository under `/opt` and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory
  
- `git clone git@github.com:nisabek/icinga2-slack-notifications.git /etc/icinga2/conf.d/`
+```
+# git clone https://github.com/nisabek/icinga2-slack-notifications.git /opt/icinga2-slack-notifications
```
Comment (nisabek) [inline README.md:91]: well, it's everyones personal choice to do https or ssh...

**Review by nisabek — COMMENTED** [review]
thanks for contribution , please find my comments.

File: README.md (hunk 1 above)
Comment (C0rn3j) [reply README.md:88]: Cloning the entire `icinga2-slack-notifications` repository when you only really want `icinga2-slack-notifications/src/slack-notifications` seems counterproductive to me, yes, as you will have to do moving anyway and then you'll have to deal with deleting the rest of the repo files unless you want icinga to try parse than as configs.

File: README.md (hunk 3 above)
Comment (C0rn3j) [reply README.md:91]: You're completely right, I've had some weird issue where git repo refused to be cloned due to lack of permissions (???) and just switched to https as a result, probably some weird temporary github issue

Comment (C0rn3j) [reply README.md]: Reverted

File: README.md (hunk 2 above)
Comment (C0rn3j) [reply README.md:73]: To denote that it is meant to run in root shell/with sudo perms, as opposed to user ($)

I really just wanted to differentiate shell commands from file examples, as both that and templates are in the readme and it was a tad confusing without it

Comment (nisabek) [reply README.md:73]: yeah, I'm just a bit worried that that's how you comment in a shell file...and who knows how one runs the commands...from a file or as a user...

File: README.md (hunk 1 above)
Comment (nisabek) [reply README.md:88]: how about this formulation:
"Clone the repository and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory" ?

Comment (C0rn3j) [reply README.md]: Sounds good, committed

File: README.md (hunk 2 above)
Comment (C0rn3j) [reply README.md]: I've switched to the `console` markdown template and made it clear it is a shell prompt.

I've also highlighted the rest of the blocks as `php`, while not being technically correct, it does highlight the text blocks nearly perfectly.

**Review by nisabek — APPROVED** [review]

**Pull request merged**